manifests/base/kustomization.yaml

```diff
@@ -5,7 +5,7 @@ commonLabels:
   app.kubernetes.io/part-of: odh-dashboard
 resources:
   - ../apps
-  - ../modelserving
+  - ../crd
```

Comment:
do you still deploy OVMS or it is just changed from modelserving folder to crd? 
if it is no OVMS, we will need to cleanup the logic in opreator

Reply:
Is there a specific logic for OVMS? We can check that today, we are moving away that folder completely and transering ownership to the modelserving team. But we can get something in place for this PR